README.md

```diff
@@ -136,16 +136,16 @@ Before deploying, make sure you have installed Terraform and a Docker client (e.
 5. Deploy via Terraform.  
    ```bash
    # Get the latest image digests
-   export TF_VAR_run_ingestion_image_path=$(gcloud container images describe gcr.io/<project-id>/<your-ingestion-image-name> \
-   --format="value(image_summary.fully_qualified_digest)")
-   export TF_VAR_run_gcs_to_bq_image_path=$(gcloud container images describe gcr.io/<project-id>/<your-gcs-to-bq-image-name> \
-   --format="value(image_summary.fully_qualified_digest)")
+   export TF_VAR_run_ingestion_image_name=$(gcloud container images describe gcr.io/<project-id>/<your-ingestion-image-name> \
+   --format="value(image_summary.digest)")
+   export TF_VAR_run_gcs_to_bq_image_name=$(gcloud container images describe gcr.io/<project-id>/<your-gcs-to-bq-image-name> \
+   --format="value(image_summary.digest)")
    
    # Deploy via terraform, providing the paths to the latest images so it knows to redeploy
-   terraform apply -var="run_ingestion_image_path=$TF_VAR_run_ingestion_image_path" \
-   -var="run_gcs_to_bq_image_path=$TF_VAR_run_gcs_to_bq_image_path
+   terraform apply -var="ingestion_image_name=<your-ingestion-image-name>@$TF_VAR_run_ingestion_image_name" \
+   -var="gcs_to_bq_image_name=<your-gcs-to-bq-image-name>@$TF_VAR_run_gcs_to_bq_image_path
    ```
```

Comment:
TF_VAR_run_gcs_to_bq_image_path also needs to be updated.

Maybe we should rename these to take out the "run_" since you removed it from the variable names? 

Also could you add the missing close quote here? Thanks!